src/ZVal.cc

```diff
@@ -0,0 +1,357 @@
+// See the file "COPYING" in the main distribution directory for copyright.
+
+#include "Val.h"
+
+#include "OpaqueVal.h"
+#include "ZeekString.h"
+#include "File.h"
+#include "Func.h"
+#include "Reporter.h"
+#include "Desc.h"
+
+using namespace zeek;
+
+
+bool* zeek::zval_error_addr = nullptr;
```

Comment:
As a small side-note. I think since c++17 you can specify zval_error_addr as inline in the header file, which lets you do the assignment directly in the header file. Using that you can get rid of the additional line in a .cc file.

Reply:
I think I answered this somewhere else, but if not: thanks for the tip!  I've made this change.